src/APIcalls.js

```diff
@@ -0,0 +1,18 @@
+const allMoviesData = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+  .then(response => response.json())
```

Comment:
This is where you can check for errors coming from the server. Check out `response.ok` property to be able to detect certain errors.